### ISSUE TITLE
the training result shows NAN! yolov4-p5

### ISSUE THREAD

**Issue by mokarms** [WongKinYiu/ScaledYOLOv4#143]
![image](https://user-images.githubusercontent.com/77577681/105308558-6e2c1600-5bc5-11eb-8455-4b980e0537a4.png)

**Comment by WongKinYiu** [comment]
adam need lower learning rate, better to 3e-4\~1e-3.
you could change it in yaml file of hyperparameter.

**Comment by mokarms** [comment]
ty @WongKinYiu 
it happend also without adam optimizer

**Comment by fede-qopius** [comment]
[https://github.com/WongKinYiu/ScaledYOLOv4/issues/145](url)

**Comment by mokarms** [comment]
@fede-qopius you know why cls  always 0

**Comment by fede-qopius** [comment]
@mokarms  difficult to know with so few information

**Comment by WongKinYiu** [comment]
if there is only one class, cls loss will always be zero.
